c-dependencies/js-compute-runtime/js-compute-builtins.cpp

```diff
@@ -5451,6 +5474,437 @@ bool constructor(JSContext *cx, unsigned argc, Value *vp) {
 }
 } // namespace TextDecoder
 
+#ifdef ENABLE_OBJECT_STORE
+namespace ObjectStoreEntry {
+namespace Slots {
+enum {
+  Body = RequestOrResponse::Slots::Body,
+  BodyStream = RequestOrResponse::Slots::BodyStream,
+  HasBody = RequestOrResponse::Slots::HasBody,
+  BodyUsed = RequestOrResponse::Slots::BodyUsed,
```

Comment:
These slots are used in the methods text/json/arraybuffer and the getters body/bodyUsed